images/tasks/Kernel.yml

```diff
@@ -0,0 +1,111 @@
+# yaml-language-server: $schema=https://taskfile.dev/schema.json
+version: '3'
```

Comment:
Would not make it generic, as it will be different for other images&archs

Reply:
I don't get the comment.